client/src/pages/Location.js

```diff
@@ -40,7 +40,13 @@ export default function Location() {
   });
   if (loading) return <Spinner />;
   if (error) return <Error error={error.message} />;
-  const {name, description, photo, reviewsForLocation: reviews, overallRating} = data?.location;
+  const {
+    name,
+    description,
+    photo,
+    reviewsForLocation: reviews,
+    overallRating
+  } = data?.location;
```

Comment:
this might just be a prettier thing on our side but it makes it annoying to launch the client app and have it throw an error